python/graphstorm/model/embed.py

```diff
@@ -107,6 +107,19 @@ def get_sparse_params(self):
         # By default, there is no sparse_embeds
         return []
 
+    def require_cache_embed(self):
```

Comment:
Can you also change GSLMNodeEncoderInputLayer and GSPureLMNodeInputLayer with require_cache_embed() returning True.

Reply:
they are changed.